src/applications/disability-benefits/all-claims/tests/all-claims.cypress.spec.js

```diff
@@ -0,0 +1,103 @@
+import path from 'path';
+
+import testForm from 'platform/testing/e2e/cypress/support/form-tester';
+import { createTestConfig } from 'platform/testing/e2e/cypress/support/form-tester/utilities';
+
+import formConfig from '../config/form';
+import manifest from '../manifest.json';
+
+import {
+  mockUser,
+  mockItf,
+  mockDocumentUpload,
+  mockPaymentInformation,
+  mockApplicationSubmit,
+} from './all-claims.cypress.helpers.js';
+
+const testConfig = createTestConfig(
+  {
+    dataPrefix: 'data',
+    dataSets: ['maximal-test', 'minimal-test', 'newOnly-test'],
+    fixtures: {
+      data: path.join(__dirname, 'data'),
+    },
+    pageHooks: {
+      introduction: () => {
+        // Hit the start button
+        cy.findAllByText(/start/i, { selector: 'button' })
+          .first()
+          .click();
+
+        // Click past the ITF message
+        cy.findByText(/continue/i, { selector: 'button' }).click();
+      },
+      'disabilities/rated-disabilities': () => {
+        cy.get('@testData').then(data => {
+          data.ratedDisabilities.forEach((disability, index) => {
+            if (disability['view:selected']) {
+              cy.get(`input[name="root_ratedDisabilities_${index}"]`).click();
+            }
+          });
+          cy.findByText(/continue/i, { selector: 'button' }).click();
+        });
+      },
+      'payment-information': () => {
+        cy.get('@testData').then(data => {
+          if (data['view:bankAccount']) {
+            cy.get('form.rjsf').then($form => {
+              const editButton = $form.find('.usa-button-primary.edit-button');
+              if (editButton) editButton.click();
+            });
+
+            cy.fillPage();
+            cy.findByText(/save/i, { selector: 'button' }).click();
+          }
+
+          cy.findByText(/continue/i, { selector: 'button' }).click();
+        });
+      },
+    },
+    setupPerTest: () => {
+      // Set up signed in session.
+      window.localStorage.setItem('hasSession', true);
+
+      // Stub API endpoints.
+      cy.route('GET', '/v0/user', mockUser)
```

Comment:
Do you think it would be useful to have a setting that handles signing in to a form. Maybe something that takes in a parameter of themockUser, and handles making the route? For example, something:

```javascript
{ ...testConfig,
   logIn(mockUser),
   ...
})
}
```

Reply:
I think so. I will add a `login` command before merging this PR.